src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -664,9 +665,22 @@ export default class AircraftCommander {
     runIls(aircraft, data) {
         const approachType = 'ils';
         const runwayName = data[1].toUpperCase();
-        const runway = AirportController.airport_get().getRunway(runwayName);
+        const runwayModel = AirportController.airport_get().getRunway(runwayName);
+        const glideslopeElevation = runwayModel.getGlideslopeAltitudeAtFinalApproachFix();
+        const minmiumGlideslopeInterceptAltitude = _ceil(glideslopeElevation, -2);
```

Comment:
I feel like the runway should simply supply this property

Reply:
done